src/tests/Navbar.test.js

```diff
@@ -0,0 +1,17 @@
+import { BrowserRouter as Router } from 'react-router-dom';
+import React from 'react';
+import renderer from 'react-test-renderer';
+import { Provider } from 'react-redux';
+import Navbar from '../components/Navbar';
+import store from '../redux/configureStore';
+
+it('matches snapshot', () => {
+  const tree = renderer.create(
+    <Provider store={store}>
+      <Router>
+        <Navbar />
+      </Router>
+    </Provider>,
+  ).toJSON();
+  expect(tree).toMatchSnapshot();
+});
```

Comment:
- [ ] At the moment, you only wrote snapshot tests. Please as pointed out by the previous reviewer, you need to include more snapshot/unit tests and integration tests.